server/functions.py

```diff
@@ -0,0 +1,21 @@
+from databases import Database
+import random
+import asyncio
+async def set_otp(username,password):
+    cookie = str(random.randint(1000000,9999999))
+    check = True
+    while check:
+        async with Database("sqlite+aiosqlite:///database.db") as database:
```

Comment:
Why connect with databases multiple times? You can connect once and use the same connection until fastapi server is running. Search: Fastapi lifecycle